Krypt/Source/SHA256.swift

```diff
@@ -11,9 +11,10 @@ import Foundation
 public struct SHA256 {
   public static func digest(_ data: Data) -> Data {
     var hash = [UInt8](repeating: 0, count: Int(CC_SHA256_DIGEST_LENGTH))
-    data.withUnsafeBytes {
-      _ = CC_SHA256($0, CC_LONG(data.count), &hash)
+    data.withUnsafeBytes { ptr in
+        guard let pointer = ptr.baseAddress?.assumingMemoryBound(to: UnsafeRawBufferPointer.self ) else { return }
```

Comment:
This one also seems to return just bytes of 0 instead of the hash in case of failed pointer conversion.